packages/circuit-ui/components/Dialog/Dialog.module.css

```diff
@@ -0,0 +1,139 @@
+.dialog {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 0;
+  margin: auto;
+  overflow: scroll;
+  pointer-events: none;
+  visibility: hidden;
+  border: none;
+  border-radius: var(--cui-border-radius-mega);
+  outline: none;
+  box-shadow: 0 3px 8px 0 rgb(0 0 0 / 20%);
```

Comment:
The modal shouldn't have a shadow according to the [Figma specs](https://www.figma.com/design/2yHC0dK9pFEhLsMxk62XZY/Circuit-UI-Web?m=auto&node-id=30195-20575&t=MDl1aiOVfwnGeis6-1).

Reply:
I initially added this for non modal dialog but forgot to remove it, good catch